From 92533301876cd45f4b5ec0f36c1f39c49575b428 Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Thu, 20 Jun 2019 20:27:26 +0300
Subject: [PATCH] ARM: dts: tegra20-glide: Remove combined-power-req property

This board uses separate-power-req, however, this property is unused
in the driver.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 arch/arm/boot/dts/tegra20-glide.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/tegra20-glide.dts b/arch/arm/boot/dts/tegra20-glide.dts
index fc4aca8a1c47..7d3b6066ac0c 100644
--- a/arch/arm/boot/dts/tegra20-glide.dts
+++ b/arch/arm/boot/dts/tegra20-glide.dts
@@ -1245,7 +1245,6 @@
 		nvidia,core-pwr-off-time = <0>;
 		nvidia,core-power-req-active-high;
 		nvidia,sys-clock-req-active-high;
-		nvidia,combined-power-req;
 		nvidia,lp0-vec = <0x1819E000 8192>;
 	};
 
-- 
2.22.0

